docker/incremental_load.sh.tpl

```diff
@@ -24,7 +24,7 @@ DOCKER="${DOCKER:-docker}"
 
 # Fetch the diff ids of the layers loaded in the docker daemon already
 IMAGES=$(${DOCKER} inspect $(${DOCKER} images -aq) |& \
-  grep sha256 | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
+  grep -E '^ +"sha256:' | grep -v Id | cut -d'"' -f 2 | cut -d':' -f 2 | \
```

Comment:
In lieu of requiring `jq` to explicitly select the `Layers` values, this looks like it will work 👍